executor/src/witgen/jit/variable.rs

```diff
@@ -42,11 +55,24 @@ impl Variable {
                 id: cell.id,
                 ptype: PolynomialType::Committed,
             }),
-            Variable::Param(_) => None,
+            Variable::Param(_) | Variable::MachineCallReturnValue(_) => None,
         }
     }
 }
 
+#[derive(PartialEq, Eq, PartialOrd, Ord, Hash, Clone, Debug)]
+pub struct MachineCallReturnVariable {
+    pub identity_id: u64,
+    pub row_offset: i32,
+    pub index: usize,
+}
+
+impl MachineCallReturnVariable {
+    pub fn into_argument<T: FieldElement>(self) -> MachineCallArgument<T, Variable> {
+        MachineCallArgument::Unknown(Variable::MachineCallReturnValue(self))
+    }
```

Comment:
For consistency, we should call it parameter instead of argument, right? (I know it was called `MachineCallArgument` before this PR.)

Reply:
Hm, I think this is fine, because we are at the calling side here, aren't we?